backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5750710_sys_gh20510_Invoice_Candidate_DeliveryStatusColor.sql

```diff
@@ -0,0 +1,198 @@
+;
+
+-- 2025-04-02T14:55:57.616Z
+INSERT INTO AD_Column_Trl (AD_Language,AD_Column_ID, Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Column_ID, t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Column t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y') AND t.AD_Column_ID=589828 AND NOT EXISTS (SELECT 1 FROM AD_Column_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Column_ID=t.AD_Column_ID)
+;
+
+-- 2025-04-02T14:55:57.719Z
+/* DDL */  select update_Column_Translation_From_AD_Element(583563) 
+;
+
+-- Column: C_Invoice_Candidate.DeliveryStatusColor_ID
+-- Column: C_Invoice_Candidate.DeliveryStatusColor_ID
+-- 2025-04-02T14:56:17.219Z
+UPDATE AD_Column SET AD_Reference_ID=18, IsExcludeFromZoomTargets='Y',Updated=TO_TIMESTAMP('2025-04-02 15:56:17','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=589828
+;
+
+-- Column: C_Invoice_Candidate.DeliveryStatusColor_ID
+-- Column: C_Invoice_Candidate.DeliveryStatusColor_ID
+-- 2025-04-02T15:09:48.885Z
+UPDATE AD_Column SET AD_Reference_ID=27,Updated=TO_TIMESTAMP('2025-04-02 16:09:48','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=589828
+;
+
+-- Column: C_Invoice_Candidate.DeliveryStatusColor_ID
+-- Column SQL (old): (CASE WHEN C_Invoice_Candidate.QtyDelivered = 0 THEN 1000000 WHEN (C_Invoice_Candidate.QtyDelivered > 0 AND C_Invoice_Candidate.QtyDelivered < C_Invoice_Candidate.QtyOrdered) THEN 1000003 WHEN (C_Invoice_Candidate.QtyDelivered = C_Invoice_Candidate.QtyOrdered) THEN 1000001 WHEN (C_Invoice_Candidate.QtyDelivered > C_Invoice_Candidate.QtyOrdered) THEN 1000004 END)
+-- Column: C_Invoice_Candidate.DeliveryStatusColor_ID
+-- Column SQL (old): (CASE WHEN C_Invoice_Candidate.QtyDelivered = 0 THEN 1000000 WHEN (C_Invoice_Candidate.QtyDelivered > 0 AND C_Invoice_Candidate.QtyDelivered < C_Invoice_Candidate.QtyOrdered) THEN 1000003 WHEN (C_Invoice_Candidate.QtyDelivered = C_Invoice_Candidate.QtyOrdered) THEN 1000001 WHEN (C_Invoice_Candidate.QtyDelivered > C_Invoice_Candidate.QtyOrdered) THEN 1000004 END)
+-- 2025-04-02T15:57:24.984Z
+UPDATE AD_Column SET ColumnSQL='(CASE WHEN C_Invoice_Candidate.QtyDelivered = 0 THEN 1000000 WHEN (C_Invoice_Candidate.QtyDelivered > 0 AND C_Invoice_Candidate.QtyDelivered < C_Invoice_Candidate.QtyOrdered) THEN 1000003 WHEN (C_Invoice_Candidate.QtyDelivered = C_Invoice_Candidate.QtyOrdered) THEN 1000001 WHEN (C_Invoice_Candidate.QtyDelivered > C_Invoice_Candidate.QtyOrdered) THEN 540006 END)',Updated=TO_TIMESTAMP('2025-04-02 16:57:24','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=589828
```

Comment:
pls do not add IDs in here; if we change something, we won't know why it broke.
So, please create a function like ```C_Invoice_Candidate_DeliveryStatus_Color_ID_Compute``` and here you will have all that togic.
But, pls do not use IDs in this function.
See that we have this function getColor_ID_By_SysConfig and you can get the color by name.
Pls use it!

Similar for InvoicingStatusColor_ID